### PR TITLE
feat(presentation): Add component LinkText

### PR DESCRIPTION
**Pull request by agory**

### CODE REVIEW

File: src/web/components/LinkText/LinkText.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { LinkText } from './';
+
+describe('Text', () => {
```
Comment (mfrachet) [inline src/web/components/LinkText/LinkText.test.js]: Bad test name

File: src/web/components/LinkText/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+LinkText.propTypes = {
+  children: PropTypes.string.isRequired,
+  className: PropTypes.string,
+  href: PropTypes.string.isRequired,
+};
+
+LinkText.defaultProps = {
+  className: '',
```
Comment (mfrachet) [inline src/web/components/LinkText/index.js]: No default classname related to bulma ?

Comment (agory) [reply src/web/components/LinkText/index.js]: It doesn't seem to me that bulma has a special class for link

**Comment by agory** [comment]
I have updated the pr :smile:

**Pull request merged**